src/legacy/core_plugins/console/np_ready/public/plugin.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import { PluginInitializerContext, Plugin, CoreStart } from '../../../../../core/public';
+import { XCoreSetup, XPluginSet } from './legacy';
+import { boot } from './app';
```

Comment:
[By convention](https://github.com/elastic/kibana/blob/master/src/core/CONVENTIONS.md), this should live in a module called "application"